content/src/quick-actions/template.md

```diff
@@ -0,0 +1,44 @@
+---
+notesMd: notes
+name: Amend Your Business Registration Certificate
+filename: template
+id: brc-amendment
+urlSlug: template-url
+callToActionLink: https://www16.state.nj.us/TYTR_REGC/servlet/common/OwnershipLogin
+callToActionText: Amend My BRC
+form: REG-C-L
```

Comment:
[pebble] My understanding is that this is a placeholder file, just to ensure that the new directory exists and gets pushed to github.

With that in mind, I don't think we should include "real" content in this file. I think at best, it's confusing (because no one is expected to look at / maintain this file) and at worst, could break future behavior (because it has a real-looking ID, which means that if another file gets created with the same ID, things could break)

I think we should make every single field in this file explicitly fake/template values instead of things that look real.

Reply:
fixed